sources/vcdMaker/src/vcdMaker.cpp

```diff
@@ -45,15 +46,36 @@ int main(int argc, const char *argv[])
     // Source registry.
     SIGNAL::SourceRegistry registry;
 
+    // Line counter.
+    INSTRUMENT::LineCounter *lineCounter = NULL;
+
     try
     {
-        // Parse the log file.
+        // Create the log parser.
         PARSER::TxtParser txtLog(cli.GetInputFileName(),
                                  cli.GetTimebase(),
                                  registry,
-                                 cli.GetLineCounterName(),
                                  cli.IsVerboseMode());
 
+        if (cli.GetLineCounterName().size())
+        {
+            // Register the line counting instrument.
+            lineCounter = new INSTRUMENT::LineCounter(cli.GetInputFileName(),
```

Comment:
I don't like the idea. Create dynamically, pass as reference, then again store as poiner. Either use `std::unique_ptr` or `std::ref` or maybe even both...